src/storage/engine/raftkv.rs

```diff
@@ -182,9 +182,6 @@ impl<C: PdClient> Engine for RaftKv<C> {
                    mut modifies: Vec<Modify>,
                    cb: Callback<()>)
                    -> engine::Result<()> {
-        if modifies.is_empty() {
-            return Ok(());
```

Comment:
`cb` is supposed to be called in another thread, calling `cb` here will break the behavior and may lead to unexpected result.